integration/helpers/base_test.py

```diff
@@ -136,11 +139,12 @@ def create_and_verify_stack(self, file_name, parameters=None):
         parameters : list
             List of parameters
         """
+        folder, file_name = file_name.split("/")
         self.output_file_path = str(Path(self.output_dir, "cfn_" + file_name + ".yaml"))
```

Comment:
Shall we also use folder in the output path? Just to ensure we will never have a collision between single and combination tests when we run them in parallel.

Reply:
I think it should be ok since all our tests have unique names.

Reply:
For now yes but this was more a comment about possible future collisions.

Reply:
Can you please add the `folder_name + "_"` before `file_name`? This way we ensure that we will never have collisions between the single and combination files.

Reply:
Added